ts/Core/Axis/DateTimeAxis.ts

```diff
@@ -26,14 +28,19 @@ declare module '../Series/SeriesOptions' {
     }
 }
 
+declare module './TickPositionsArray' {
+    interface TickPositionsArray extends Array<number> {
+    }
+}
```

Comment:
This should be part of the TickPositionsArray file, because it is the essential definition of the type.